ci/pipeline.yml

```diff
@@ -72,22 +72,21 @@ jobs:
       trigger: true
     - get: nfs-volume-release
       trigger: true
-    - get: common-development
-      trigger: true
     - get: stemcell
       trigger: true
     - get: master-bosh-root-cert
   - put: nfs-server-development
-    params: &nfs-server-params
+    params:
```

Comment:
Why drop the yaml anchor here and re-add it below?

Reply:
To keep the work on development islolated until we have feature parity with CH in all envs in the future.

Reply:
I mean this is only needed cause of the ops file though.